glide.yaml

```diff
@@ -0,0 +1,76 @@
+package: github.com/openshift/origin
+excludeDirs:
+- cmd/cluster-capacity
+- cmd/service-catalog
+- pkg/build/builder
+import:
+- package: github.com/coreos/etcd
+  version: v3.2.8
+- package: github.com/gonum/graph
+  version: bde6d0fbd9dec5a997e906611fe0364001364c41
+- package: github.com/google/certificate-transparency
+  repo:    git@github.com:openshift/google-certificate-transparency
+  version: master
+- package: github.com/vishvananda/netlink
+  version: 933b978eae8c18daa1077a0eb7186b689cd9f82d
+- package: github.com/google/cadvisor
+  repo:    git@github.com:openshift/google-cadvisor
+  version: release-0.27.1
+- package: github.com/emicklei/go-restful-swagger12
+  repo:    https://github.com/openshift/emicklei-go-restful-swagger12
+  version: release-1.0.1
+- package: google.golang.org/appengine
```

Comment:
that are more than I expected. It's the minimal set? Would be great to have a comment with the reason why we chose another version than upstream.

Reply:
> that are more than I expected. It's the minimal set? Would be great to have a comment with the reason why we chose another version than upstream.

Explanations as a followup as we try to prune.  This is the minimal set for now.